packages/common/mui/src/lib/info-tooltip/info-tooltip.tsx

```diff
@@ -27,12 +28,14 @@ export function InfoTooltip({
   sx,
   iconColor,
   iconSize,
+  onMouseEnter,
```

Comment:
Same here question here as the `Faq` component, can we just import `trackEvent` here instead?